full/full_coupler_mod.F90

```diff
@@ -1503,5 +1523,107 @@ subroutine ocean_chksum(id, timestep, Ocean, Ice_ocean_boundary)
 
   end subroutine ocean_chksum
 
+!> \brief This subroutine sets the ID for clocks used in coupler_main
+  subroutine coupler_set_clock_ids(coupler_clocks, Atm, Land, Ice, Ocean, ensemble_pelist,&
```

Comment:
This could be a type bound procedure.  This is the setter.  Please add it to the issue you open.